examples/parallel-prepared.rs

```diff
@@ -28,6 +28,7 @@ async fn main() -> Result<()> {
             .prepare("INSERT INTO ks.t2 (a, b, c) VALUES (?, ?, 'abc')")
             .await?,
     );
+    println!("Prepared statement: {:?}", prepared);
```

Comment:
```suggestion
    println!("Prepared statement: {:#?}", prepared);
```
The example would be nicer if it did pretty debug print.
Instead of a cryptic one-liner: 
```
prepared statement: PreparedStatement { config: StatementConfig { consistency: None, serial_consistency: Some(LocalSerial), is_idempotent: false, retry_policy: None, speculative_execution_policy: None, tracing: false, timestamp: None }, prepare_tracing_ids: [], id: b"#\xc9\xcd\x85\x1b\xa9\xa5Q\xc1;\x15\xc8X+\x0c\xb5", metadata: PreparedMetadata { col_count: 2, pk_indexes: [PartitionKeyIndex { index: 0, sequence: 0 }], col_specs: [ColumnSpec { table_spec: TableSpec { ks_name: "ks", table_name: "t2" }, name: "a", typ: Int }, ColumnSpec { table_spec: TableSpec { ks_name: "ks", table_name: "t2" }, name: "b", typ: Int }] }, statement: "INSERT INTO ks.t2 (a, b, c) VALUES (?, ?, 'abc')", page_size: None, partitioner_name: Murmur3 }
```
This would produce neat readable output:
```
Prepared statement: PreparedStatement {
    config: StatementConfig {
        consistency: None,
        serial_consistency: Some(
            LocalSerial,
        ),
        is_idempotent: false,
        retry_policy: None,
        speculative_execution_policy: None,
        tracing: false,
        timestamp: None,
    },
    prepare_tracing_ids: [],
    id: b"#\xc9\xcd\x85\x1b\xa9\xa5Q\xc1;\x15\xc8X+\x0c\xb5",
    metadata: PreparedMetadata {
        col_count: 2,
        pk_indexes: [
            PartitionKeyIndex {
                index: 0,
                sequence: 0,
            },
        ],
        col_specs: [
            ColumnSpec {
                table_spec: TableSpec {
                    ks_name: "ks",
                    table_name: "t2",
                },
                name: "a",
                typ: Int,
            },
            ColumnSpec {
                table_spec: TableSpec {
                    ks_name: "ks",
                    table_name: "t2",
                },
                name: "b",
                typ: Int,
            },
        ],
    },
    statement: "INSERT INTO ks.t2 (a, b, c) VALUES (?, ?, 'abc')",
    page_size: None,
    partitioner_name: Murmur3,
}
```